App/src/main/java/cc/softwarefactory/lokki/android/activities/MainActivity.java

```diff
@@ -79,10 +88,31 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        mNavigationDrawerFragment = (NavigationDrawerFragment) getSupportFragmentManager().findFragmentById(R.id.navigation_drawer);
         mTitle = getTitle();
+
+        mNavigationDrawerFragment = (NavigationDrawerFragment) getSupportFragmentManager().findFragmentById(R.id.navigation_drawer);
         mNavigationDrawerFragment.setUp(R.id.navigation_drawer, (DrawerLayout) findViewById(R.id.drawer_layout)); // Set up the drawer.
 
+        LayoutInflater inflater = getLayoutInflater();
+        ListView drawerListView = (ListView) ((FrameLayout)mNavigationDrawerFragment.getView()).getChildAt(0);
+        infoView = inflater.inflate(R.layout.fragment_drawer_header, drawerListView, false);
+
+        setUserInfo();
+
+        drawerListView.addHeaderView(infoView);
+    }
+
+
+    public void setUserInfo() {
+        AQuery aq = new AQuery(this, infoView);
+
+        ImageView avatarImage = (ImageView) infoView.findViewById(R.id.avatar);
+        AvatarLoader avatarLoader = new AvatarLoader(this);
+
+        String email = PreferenceUtils.getValue(this, PreferenceUtils.KEY_USER_ACCOUNT);
```

Comment:
Do we still need this method? Maybe refactor the method name to `getString` as it is using this method to get the prefs?